src/modules/dashboard/components/TokenMintDialog/TokenMintDialog.tsx

```diff
@@ -35,79 +39,113 @@ interface FormValues {
   mintAmount: number;
 }
 
-interface Props {
-  cancel: () => void;
-  close: () => void;
-  nativeToken: ColonyTokens[0];
-  colonyAddress: Address;
+interface CustomWizardDialogProps {
+  prevStep?: string;
+  colony: Colony;
 }
 
+type Props = DialogProps &
+  Partial<WizardDialogType<object>> &
+  CustomWizardDialogProps;
+
+const displayName = 'dashboard.TokenMintDialog';
+
 const TokenMintDialog = ({
```

Comment:
These two files are all mixed up!

`TokenMintDialog` contains the form and `TokenMintDialogForm` contains the dialog.

Please swap the contents of these two files around.

Reply:
Im lost :D I see only one dialog which is parent of TokenMintForm

Reply:
So inside the `TokenMintDialog` component **folder** there are two `jsx` files: 
- `TokenMintDialog`
- `TokenMintForm`

![Screenshot from 2021-01-05 14-25-36](https://user-images.githubusercontent.com/1193222/103646223-ff628200-4f61-11eb-8850-ec18c34cfe22.png)

The "convention" is that the "Dialog" file contains the "wrapper" component files _(the part that contains the `ActionForm`)_, while the other file, the "Form" contains the actual form and dialog component elements _(the part that has all the `DialogSection`s components)_

You can see and example of this with the `TransferFundsDIalog` component: https://github.com/JoinColony/colonyDapp/tree/feature/dev-65-mint-tokens-flow-wiring/src/modules/dashboard/components/TransferFundsDialog

In your case here, they are reversed, the dialog file contains the dialog sections, while the "form" contains the wrapper.

Just swap the content of these two files around, and it should be all good.

_PS: Let me know after you've made this change so I can re-review this. Thanks_